src/code.sh

```
@@ -0,0 +1,58 @@
+
+# download all inputs, untar plug-n-play resources, and set PATH to arriba directory
+# mark-section "Download inputs and set up initial directories and values"
+tar -xzf /home/dnanexus/in/arriba_tar/*.tar.gz -C /home/dnanexus/
+cd arriba_v2.4.0 && make
+cd ..
+export PATH=$PATH:/home/dnanexus/arriba_v2.4.0
+
+bams=($(ls /home/dnanexus/input_bams*.bam))
+
+#Extract sample name from input_bam
+sample_name=$(echo $bams[0] | cut -d '_' -f 1)
```

Shellcheck (0.10.0)

[error] Use braces when expanding arrays, e.g. ${array[idx]} (or ${var}[.. to quiet).

(SC1087)